gratipay/testing/harness.py

```diff
@@ -211,17 +211,19 @@ def make_admin(self, username='admin'):
 
 
     def make_package(self, package_manager=NPM, name='foo', description='Foo fooingly.',
-                                                      emails=['alice@example.com'], claimed=False):
+                                                    emails=['alice@example.com'], claimed_by=None):
         """Factory for packages.
         """
         self.db.run( 'INSERT INTO packages (package_manager, name, description, emails) '
                      'VALUES (%s, %s, %s, %s) RETURNING *'
                    , (package_manager, name, description, emails)
                     )
         package = Package.from_names(NPM, name)
-        if claimed:
+        if claimed_by:
+            if claimed_by == 'picard':
```

Comment:
... Why hardcode `picard` here? Couldn't we just check if the participant exists and create it if not (like the other similar functions)